### PR TITLE
Release/1.0.0 - document restructuring 

### PR DESCRIPTION
**Pull request by cs-suyog-jinturkar**
document restructuring 

UTC:
1. Verified that all document links are working as expected
2. Verified that all document content is displayed correctly.

### CODE REVIEW

File: docs/contents.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+|Alerts SLA Notification|Notify users when SLA is about to be breached|
+|Incidents SLA Notification|Notify users when SLA is about to be breached|
+|Send Message to Teams Channel|Send message to channel|
+
+>**Warning:** We recommend that you clone these playbooks before customizing to avoid loss of information while upgrading the solution pack.
+
+- Dashboard:
+   - SLA Monitor
+
+- Configuration step **@Send Message to Teams Channel** playbook
```
Comment (cs-sujeet-ahirrao) [inline docs/contents.md:24]: Also same for Alert

File: docs/setup.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+1. To install a solution pack, click **Content Hub** > **Discover**.
+2. From the list of solution pack that appears, search for and select **SLA Notification**.
+3. Click the **SLA Notification** solution pack card.
+4. Click **Install** on the bottom to begin installation.
+
+## Prerequisites
+**Not Applicable**
+
+## Configuration:
+- Configuration step **@Incidents SLA Notification:** playbook
```
Comment (cs-sujeet-ahirrao) [inline docs/setup.md:15]: Same for Alert playbook

File: docs/usage.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+| [Home](../README.md) |
+|--------------------------------------------|
+
+# Usage
+
+Run the playbook: Create Alerts and Incidents Records to populate FortiSOAR with sample records, then browse to SLA Notification Dashboard
```
Comment (cs-sujeet-ahirrao) [inline docs/usage.md]: Playbook name is required

File: docs/setup.md (hunk 2 above)
Comment (cs-suyog-jinturkar) [reply docs/setup.md:15]: NA

File: docs/usage.md (hunk 3 above)
Comment (cs-suyog-jinturkar) [reply docs/usage.md]: done

**Pull request merged**